rules/relational-database-design/rule.md

```diff
@@ -19,283 +21,9 @@ If a database lacks proper structure and organization, many problems can arise.
-
-| FirstName | LastName  | CourseName | Instructor    |
-| --------- | --------- | ---------- | ------------- |
-| Alice     | Smith     | Math       | Prof. Smith   |
-| Alice     | Smith     | English    | Prof. Johnson |
-| Bob       | Northwind | Math       | Prof. Smith   |
-
-#### ✅ Good Example - Architecting the schema to have 3 tables
-
-A better way to architect the schema for this scenario is to break it into 3 tables with 2 relationships. Now the data is properly normalized and we can track which students do which courses, and any extra details such as when the student starts the course.
-
-**Students Table**
-
-Contains the student data
-
-| StudentID | Name  | LastName  |
-| --------- | ----- | --------- |
-| 1         | Alice | Smith     |
-| 2         | Bob   | Northwind |
-
-**Courses Table**
-
-Contains the course data
-
-| CourseID | CourseName | Instructor    |
-| -------- | ---------- | ------------- |
-| 1        | Math       | Prof. Smith   |
-| 2        | English    | Prof. Johnson |
-
-**StudentCourses Table**
-
-Contains the data about which students are taking which courses
-
-| StudentID | CourseID |
-| --------- | -------- |
-| 1         | 1        |
-| 1         | 2        |
-| 2         | 1        |
-
-**Relationship 1 - Students and StudentCourses**
-- 1 Student has many StudentCourses
-- 1 StudentCourse has 1 Student
-
-**Relationship 1 - Courses and StudentCourses**
-- 1 Course has many StudentCourses
-- 1 StudentCourse has 1 Course 
-
-## Visualization
-
-Note how wordy it was to illustrate the many-to-many relationship in the StudentCourses example. Now imagine that database expands to have 10 more tables. It would quickly become hard to keep track of everything. This problem is what ERDs solve.
-
-An ERD helps quickly display all the relationships in a database at a glance. Let's see what it looks like for StudentCourses:
-
-![Figure: Student Courses ERD](studentcourseserd.png)
-
-In this example, Students, Courses and StudentCourses are represented via the rectangles. Meanwhile, their relationships are shown via the lines between the rectangles. You can see the cardinality indicated by what is called [Crow's foot notation](https://en.wikipedia.org/wiki/Entity%E2%80%93relationship_model#Crow's_foot_notation).  
-
-# Summary
+## The Solution
+Solving these problems is hard. Luckily, there are some tried and tested tools that help resolve these problems including:
+- [Database Normalization](https://www.ssw.com.au/rules/database-normalization)
+- [Entity Relationship Diagrams (ERDs)](https://www.ssw.com.au/rules/erds)
 
 Database normalization and ERDs are the core tools needed to perform good relational database design. If misapplied, it can have disastrous consequences with far-reaching effects. For example, once you start putting data into a table, it becomes significantly harder to fix since you now have a data migration problem. That's why getting database design right the first time is vital.
```

Comment:
"If misapplied, **it** can have..." should read "If misapplied, **they** can have...".

Reply:
changed to:

If these concepts are misapplied or are not used at all, there can be disastrous consequences with far-reaching effects.